wellregistry/registry/tests/test_admin.py

```diff
@@ -1,140 +1,149 @@
 """
 Tests for registry admin module
 """
-import datetime
 
+from django.contrib.admin.sites import AdminSite
+from django.contrib.auth.models import User, Group
+from django.http import HttpRequest
 from django.test import TestCase
 
-from ..admin import RegistryAdminForm, RegistryAdmin, check_mark
-from .lookups import create_lookup_data
-from ..models import AgencyLookup, CountyLookup, StateLookup, Registry
+from ..admin import RegistryAdmin, check_mark
+from ..models import AgencyLookup, Registry
 
-class TestRegistryAdminForm(TestCase):
```

Comment:
After talking with Ed, we decided to remove this test because it really wasn't testing anything. The user can not select an invalid lookup so testing for invalid lookups seemed unnecessary. At this point we really don't have any custom validations that need to be tested.